resources/views/ItemList/Components/Filter/ItemFilterPrice.twig

```diff
@@ -25,7 +25,8 @@
         </div>
 
         <button type="button" class="btn btn-primary btn-appearance" :class="{'disabled': isDisabled}" v-tooltip
-                data-toggle="tooltip" data-placement="top" @click="triggerFilter()" :title="$translate('Ceres::Template.itemApply')">
+                data-toggle="tooltip" data-placement="top" @click="triggerFilter()" :title="$translate('Ceres::Template.itemApply')"
+                rel="nofollow">
```

Comment:
When you already put `rel="nofollow"` on a newline, please do this with all attributes in this element.

E.g.
```html
<button
    attr1="a"
    attr2="b"
>
```